app/services/hyrax/admin_set_service.rb

```diff
@@ -64,5 +66,17 @@ def data_attributes(admin_set)
         attrs['data-visibility'] = permission_template.visibility unless permission_template.visibility.blank?
         attrs
       end
+
+      # Count number of files from admin set works
+      # @param [Array] Solr search result
+      def count_files(results)
```

Comment:
Ah, this will be a problem because we don't have works here, we have admin sets due to https://github.com/projecthydra-labs/hyrax/blob/master/app/search_builders/hyrax/admin_set_search_builder.rb#L16.  I'm not sure you ever see file_set_ids_ssim in your results, will you?